apps/site/pages/components/molecules/carousel.mdx

```diff
@@ -0,0 +1,326 @@
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+import {
+  ProductCard,
+  ProductCardContent,
+  ProductCardImage,
+  ProductShelf,
+  ProductShelfItems,
+  ProductShelfItem,
+} from '@faststore/ui'
+
+import CarouselUsage from 'site/components/Carousel/CarouselUsage'
+import { products } from 'site/mocks/products'
+import { useFormattedPrice } from 'site/components/utilities/usePriceFormatter'
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const carouselPath = path.resolve(__filename)
+  const components = ['Carousel.tsx']
+  const [carouselProps] = getComponentPropsFrom(carouselPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: { carouselProps },
+    },
+  }
+}
+
+export const CarouselPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { carouselProps } = useSSG()
+  return {
+    Carousel: <PropsSection propsList={carouselProps} />,
+  }[component]
+}
+
+<header>
+
+# Carousel
+
+The Carousel component is great for swiping images.
+
+</header>
+
+We recommend using Carousel when displaying at least 10 items.
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { Carousel } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/molecules/Carousel/styles.scss'
+```
+
+---
+
+## Usage
+
+<Callout type="warning" emoji="⚠️">
+  This component is a work in progress, so it might work as expected in some
+  cases, we gratefully accept feedback.
+</Callout>
+
+For the mobile preview, please refresh the browser after resize it.
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection style={{ width: '100%', overflow: 'hidden' }}><CarouselUsage /></OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+    const isMobile = window.innerWidth <= 768
+
+    <Carousel
+      itemsPerPage={isMobile ? 1 : 3}
+      variant="scroll"
+      infiniteMode={false}
+    >
+      {products.map(({ product }, idx) => (
+        <ProductCard key={idx}>
+          <ProductCardImage>
+            <img
+              data-fs-image
+              src={product.image[0].url}
+              alt={product.image[0].alternateName}
+            />
+          </ProductCardImage>
+          <ProductCardContent
+            title={product.isVariantOf.name}
+            price={{
+              value: product.offers.offers[0].price,
+              listPrice: product.offers.offers[0].listPrice,
+              formatter: useFormattedPrice,
+            }}
+          />
+        </ProductCard>
+      ))}
+    </Carousel>
+
+    ```
+
+  </Tab>
+</Tabs>
+
+---
+
+## Props
+
+<CarouselPropsSection component="Carousel" />
+
+---
+
+## Design Tokens
```

Comment:
🎗️  Remember to update this list if you agree with my proposal for the arrows.

Reply:
Thanksss!